Test_Framework/UnitTest1.cs

```diff
@@ -0,0 +1,182 @@
+
+            Consecutove_Range_Of_CurrentSamples = InstanceConsecutive_Range_Check.Main_Range_Dictionary(Input1);
+            var Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["1-4"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 5);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["8-9"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 2);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["1000-1002"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 3);
+
+            /* Test case with Test sample list 2 */
+            Consecutove_Range_Of_CurrentSamples = InstanceConsecutive_Range_Check.Main_Range_Dictionary(Input2);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["1"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 1);
+
+            /* Test case with Test sample list 3 */         
+            Consecutove_Range_Of_CurrentSamples = InstanceConsecutive_Range_Check.Main_Range_Dictionary(Input3);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["2-2"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 5);
+
+            /* Test case with Test sample list 4 */         
+            Consecutove_Range_Of_CurrentSamples = InstanceConsecutive_Range_Check.Main_Range_Dictionary(Input4);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["1-3"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 3);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["10-13"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 6);
+
+
+            /* Test case with Test sample list 5 */
+            Consecutove_Range_Of_CurrentSamples = InstanceConsecutive_Range_Check.Main_Range_Dictionary(Input5);
+            Consecutive_Range_Of_CurrentSample_Range = Consecutove_Range_Of_CurrentSamples["0-0"];
+            Debug.Assert(Consecutive_Range_Of_CurrentSample_Range.Count() == 3);
+            
+        }
+
+        [Fact]
+        public void Test_method_Delete_NonCosecutive_Numbers()
+        {
+            /* Test case with Test sample list 1 */
+            Input1.Sort();
+            List<int> CurrentSamples_Without_Non_Consecutive_No = new List<int>();
+             CurrentSamples_Without_Non_Consecutive_No=InstanceConsecutive_Range_Check.Delete_NonCosecutive_Numbers(Input1.ToArray());
+             List<int> Expected_List_1 = new List<int> {1,2,2,3,4,8,9,1000,1001,1002};         
+             Debug.Assert(CurrentSamples_Without_Non_Consecutive_No.Count() == Expected_List_1.Count());
+             Help_Method(CurrentSamples_Without_Non_Consecutive_No, Expected_List_1);  
```

Comment:
It isn't possible to infer the purpose of `Help_Method`, unless i go through its implementation.
It is checking the equality of two arrays.
Why not name it 'Arrays_Are_Equal`?
This name triggers a thought that the function itself can check the length of the array. If you add it into the function itself, you can avoid repeating the length check in all the tests